src/main/java/xyz/groundx/caver_ext_kas/kas/kip7/KIP7.java

```diff
@@ -0,0 +1,1459 @@
+     * }
+     * </pre>
+     * @param addressOrAlias Contract address (in hexadecimal with the 0x prefix) or an alias.
+     * @param from Klaytn account address to send tokens.
+     * @param to Klaytn account addree to receive tokens
+     * @param amount The amount of tokens to transfer
+     * @return Kip7TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip7TransactionStatusResponse transfer(String addressOrAlias, String from, String to, BigInteger amount) throws ApiException {
+        return transfer(addressOrAlias, from, to, Numeric.toHexStringWithPrefix(amount));
+    }
+
+    /**
+     * Sends tokens of the contract asynchronously. <br>
+     * POST /v1/contract/{contract-address-or-alias}/transfer <br>
+     * <pre>Example :
+     * {@code
+     * ApiCallback<Kip7TransactionStatusResponse> callback = new ApiCallback<Kip7TransactionStatusResponse>() {
+     *     ....implements callback method.
+     * }
+     *
+     * String contractAlias = "";
+     * String owner = "";
+     * String to = "";
+     * BigInteger amount = BigInteger.valueOf(10).multiply(BigInteger.TEN.pow(18)); // 10 * 10^18
+     *
+     * caver.kas.kip7.transferAsync(contractAlias, owner, to, amount, callback);
+     * }
+     * </pre>
+     * @param addressOrAlias Contract address (in hexadecimal with the 0x prefix) or an alias.
+     * @param from Klaytn account address to send tokens.
+     * @param to Klaytn account addree to receive tokens
+     * @param amount The amount of tokens to transfer.
+     * @param callback The callback to handle response.
+     * @return Call
+     * @throws ApiException
+     */
+    public Call transferAsync(String addressOrAlias, String from, String to, BigInteger amount, ApiCallback<Kip7TransactionStatusResponse> callback) throws ApiException {
+        return transferAsync(addressOrAlias, from, to, Numeric.toHexStringWithPrefix(amount), callback);
+    }
+
+    /**
+     * Sends tokens of the contract. <br>
+     * POST /v1/contract/{contract-address-or-alias}/transfer <br>
+     * <pre>Example :
+     * {@code
+     * String contractAlias = "";
+     * String owner = "";
+     * String to = "";
+     * String amount = "0x8ac7230489e80000"; // 10 * 10^18
+     *
+     * Kip7TransactionStatusResponse response = caver.kas.kip7.transfer(contractAlias, owner, spender, amount);
+     * }
+     * </pre>
+     * @param addressOrAlias Contract address (in hexadecimal with the 0x prefix) or an alias.
+     * @param from Klaytn account address to send tokens.
+     * @param to Klaytn account addree to receive tokens
+     * @param amount The amount of tokens to transfer. (in hexadecimal with the 0x prefix)
+     * @return Kip7TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip7TransactionStatusResponse transfer(String addressOrAlias, String from, String to, String amount) throws ApiException {
```

Comment:
The `from` parameter here also can be skipped. Please check `TransferKip7TokenRequest` component. The default value of the `from` is the contract deployer account.

Reply:
I added it.